polaris-react/src/components/Grid/Grid.tsx

```diff
@@ -32,17 +32,23 @@ export interface GridProps {
   columns?: Columns;
   children?: React.ReactNode;
 }
+
+const tokenToVariable = (value: SpacingTokens) => `var(--p-${value})`;
```

Comment:
Suggest updating this to use the `createVar` utility from `@shopify/polaris-tokens`:

```js
import {createVar} from '@shopify/polaris-tokens';
```
```suggestion
const tokenToVariable = (value: SpacingTokens) => `var(${createVar(value)})`;
```